dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -91,5 +95,100 @@ interface RequestApi {
     fun getAggregatedDataRequests(
         @RequestParam identifierValue: String? = null,
         @RequestParam dataTypes: Set<DataTypeEnum>? = null,
-    ): List<AggregatedDataRequest>
+        @RequestParam reportingPeriod: String? = null,
+    ): ResponseEntity<List<AggregatedDataRequest>>
+
+    /**
+     * A method to post a single request to Dataland.
```

Comment:
A question which came up in a call with @fsoffel and @emanueldfine , is it necessary that the single request data endpoint accepts LEIs, ISINs etc or would it be sufficient if only dataland companyIds would be accepted?

Reply:
We also talked about this one before. It may have been sufficient as well. However, we thought it could be counter-intuitive for an API user that (s)he can pass an identifier (lei, isin or permId) when sending a bulk request while this is not possible in a single request. Nevertheless, since the additional feature of accepting other identifiers is already implemented (and it should not harm anything), I would vote for not changing it again. Objections @stewal19 @fsoffel @emanueldfine ?

Reply:
Problem is: For the e-mail which should be send out, we need the company name, which we can get by using the companyId. However if we only have for example a LEI i think we currently are not able to determine the company name.

Reply:
I see. But what about the bulk data request? This should also cause multiple emails to be sent, right? There we deliberately decided when setting up the community manager that requests should also be stored if the identifier could not be mapped to a company existing on Dataland... What to you think @emanueldfine ?

Reply:
Bulk data requests currently only send an email to the dataland@d-fine adress and to erik.
However, maybe shortly align with the subteam and florian here

Reply:
Currently the single request endpoint should behave in the same way as the bulk request. So i will close this here